src/binder/bind_node_visitor.cpp

```diff
@@ -18,7 +18,10 @@
 namespace peloton {
 namespace binder {
 
-BindNodeVisitor::BindNodeVisitor() { context_ = nullptr; }
+BindNodeVisitor::BindNodeVisitor(concurrency::Transaction *txn) {
```

Comment:
Use C++11 initializer lists for member initialization [like so](http://en.cppreference.com/w/cpp/language/initializer_listl). What you have isn't wrong, but it is inconsistent with the rest of the code.